hathor/p2p/manager.py

```diff
@@ -317,11 +334,29 @@ def update_whitelist(self) -> Deferred[None]:
             settings.WHITELIST_URL.encode(),
             Headers({'User-Agent': ['hathor-core']}),
             None)
+        # Twisted Agent does not have a direct way to configure the HTTP client timeout
+        # only a TCP connection timeout. The callLater below is a manual client timeout
```

Comment:
I agree that a TCP connection timeout is not the best for us. But I feel we should enhance this comment to explain that we want a timeout that encompasses the connection and download time.